cpack/debian/prerm

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+if [[ $1 == "remove" ]]; then
+    systemctl stop fluent-bit.service || true
+    systemctl disable fluent-bit.service || true
```

Comment:
Shouldn't we check whether service or systemctl command is effective on the Debian box?

Reply:
Sure! I will test it on Debian 11 and 12 and reply back with results.

Reply:
Tests have been successfully done on debian bullseye and bookworm versions.
After Debian 8 systemd is the default init system for debian ([Debian Reference](https://wiki.debian.org/systemd#Introduction)) so systemctl would not make any issues.

Reply:
Sounds great. So, we shouldn't consider the outdated Debian system. Let's leave as-is.